### DISCUSSION TITLE
`_lastRenderedLogicalSize` is assigned but never used

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#13987] [Q&A] [answered]
https://github.com/AvaloniaUI/Avalonia/blob/06a3727a6f7826ab03073eff014901bf6168f4ed/src/Avalonia.Native/WindowImplBase.cs#L61

https://github.com/AvaloniaUI/Avalonia/blob/06a3727a6f7826ab03073eff014901bf6168f4ed/src/Avalonia.Native/WindowImplBase.cs#L163-L180

Is it safe to remove or is something missing?

**Comment by maxkatz6** [comment, marked as answer]
Yes, it's not needed anymore. Previously it was used when Deferred Renderer was enabled: https://github.com/AvaloniaUI/Avalonia/commit/ce1d85599977cd88c776688c29e02c9cf6cded91
